docs/applications/server.md

```diff
@@ -31,6 +32,42 @@ This will produce a script to execute bitcoin-s which you can start with
 ./app/server/target/universal/stage/bin/bitcoin-s-server
 ```
 
+#### Docker
+
+The oracle server also has docker support. You can build a docker image with the following commands
+
+```
+sbt "appServer/compile;appServer/docker:stage"
```

Comment:
does this need the compile first, shouldn't `docker:stage` compile it

Reply:
It's not strictly necessary, i can remove if you desire

Reply:
imo it'd be cleaner without